scripts/generate-files

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
```

Comment:
Considering that this file will generate them, do we still need to have adb.json, manifest.json, package.json and package-lock.json in the repository?

Reply:
The script isn't executed by the pipeline. I added the source files to this repo to have a copy of what gets signed. If we somehow let the script generate the files on the fly in TaskCluster, I am a little worried that this might complicate things if we have to debug (pipeline-related) issues. WDYT?

Reply:
That makes sense, I wasn't sure if the script would get executed or not. 
Once the setup is working we can just update the README to mention how generate-files can be used to prepare a new release.